src/server/status/index.js

```diff
@@ -29,7 +34,8 @@ export default function (kbnServer, server, config) {
           build_snapshot: matchSnapshot.test(config.get('pkg.version'))
         },
         status: kbnServer.status.toJSON(),
-        metrics: kbnServer.metrics
+        metrics: kbnServer.metrics,
+        stats,
```

Comment:
+1 to ^ and +1 on a new API, at this point they're hitting different services.

We have the ops interval for fetching server and os information.  These are (relatively) quick reads and all have to do with how the kibana server is running.  We need this information for the status page to say how kibana's running.

Adding saved object stats will make this too chatty and less focused.  I don't think we should put the caching at this level, but at the consumer's level.  Any time someone requests stats, we can hit es and return the most recent data.